k8s/namespaces/ia/ia-case-access-api/aat.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-case-access-api
+spec:
+  values:
+    java:
+      ingressHost: ia-case-access-api-aat.service.core-compute-aat.internal
```

Comment:
can you remove the line with `ingressHost`?
we have this configuration in values.yaml in app repository

Reply:
ok